core-bundle/src/Twig/Extension/ContaoExtension.php

```diff
@@ -62,13 +62,13 @@
 use Twig\TwigFilter;
 use Twig\TwigFunction;
 
-/**
- * @experimental
- */
 final class ContaoExtension extends AbstractExtension implements GlobalsInterface
 {
     private array $contaoEscaperFilterRules = [];
 
+    /**
+     * @internal
+     */
```

Comment:
What's the point of adding this to the constructor instead of to the class?

Reply:
So that you can use the class as a service (that you get from the service container) but we can still adjust the dependencies.